guide/principles/principles.md

```diff
@@ -0,0 +1,168 @@
+- Run a full Bitcoin node when possible
+- Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}}) 
+- Offer the user choice of what node and other external services to connect to
+
+**Don't**
+- Introduce a single point of failure between the user and the Bitcoin network
+- Build products that stop working if the project shuts down
+
+---
+
+## Inclusion
+
+> Provide equal access regardless of the user's location, language, ability and prior knowledge of Bitcoin
+
+There are no background checks, credit checks, or gatekeepers to Bitcoin. A Kenyan farmer has the same access to Bitcoin as a Wall Street trader. 
+
+And while Bitcoin is already widely used by a large number of people, it pales in comparison with the many more that are likely to use it in the future. We need to design products that are prepared for people new to Bitcoin, without reducing the possibilities. This means using plain and familiar language, explaining things in the context where they are needed, not overwhelming people with technical detail, and more.
+
+**Do**
+- Provide equal and direct access to the Bitcoin network, without middlemen and exclusive requirements
+- Design Bitcoin products that are usable by the widest range of people possible
+- Use plain language that people new to Bitcoin can understand regardless of prior knowledge
+- Use more languages than just English
+- Educate in place, when people are presented with a new concept
+- Treat users with impairments, or who rely on assistive technologies, as first-class citizens
+
+**Don't**
+- Introduce requirements that might exclude many people, for example by building only for high-end hardware, requiring US KYC (Know-your-customer) information etc
+- Add technical detail that is not required knowledge, or technical terms like seed phrase XPUBs, mnemonics etc.
+- Put all education up front and expect people to read and remember it
+
+---
+
+## Transparency
+
+> Open-source your products and clearly explain how they work
+
+While an open and decentralized financial system that lets people connect directly with it is great, it does put a lot of burden on the users to pick a product they trust, and that works for them. We can make this easier by freely sharing information about how our products work, what technologies they use and rely on and so forth. Let people verify that your claims are true. It is also very important to be open about what risks the user might be taking on with self-custody, under which scenarios they might lose access to their funds and best practices for avoiding this. 
+
+**Do**
+- Be open and transparent with how your product works
+- Build your product as open-source when possible
+- Let people verify, without having to trust
```

Comment:
I think we should elaborate on the `Don't trust - verify` a bit more in the opening paragraph. The bullet-point says basically the same thing as the one in the paragraph above.

Reply:
Addressed in 661e72f